.travis.yml

```diff
@@ -3,7 +3,7 @@ jdk:
 os:
   - linux
 language: scala
-
+dist: trusty
```

Comment:
what does it do?

Reply:
specifies distribution to be ubuntu trusty - I think there is an issue when the distro is newer and it doesn't have jdk 8.

Reply:
@olafurpg does this make sense? I had issues with jdk8 before adding this, but I have no experience with travis.

Reply:
LGTM